user/urls.py

```diff
@@ -1,6 +1,10 @@
 from django.urls import path
 from user.views import CreateUserView, CreateTokenView, ManageUserView
 
+
+
+
+
 app_name = "user"
 
 urlpatterns = [
```

Comment:
interesting changes, but where are the changes you had to implement in this task? :)